### PR TITLE
1 create template for specifications

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
- [x] Create repository structure incl. Readme
- [x] Create specification template
- [x] Add Markdown linter

### CODE REVIEW

File: EVES/eves-0001.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+|EVES Purpose and Process
+
+|Type
+|Process
+
+|Linked to
+|-
+
+|Status
+|Draft
```
Comment (jdsika) [inline EVES/eves-0001.adoc:18]: Give hints about the expected input

Comment (ClemensLinnhoff) [reply EVES/eves-0001.adoc:18]: Or do you mean as a sort of comment? I have to check if this is possible in AsciiDoc.

**Comment by ClemensLinnhoff** [comment]
Apparently there is no real out-of-the-box adoc linter...

**Comment by jdsika** [comment]
The linter output is intense but let's keep it that way for now

**Pull request merged**